lib/srv/app/aws/handler.go

```diff
@@ -115,7 +120,7 @@ func NewAWSSignerHandler(ctx context.Context, config SignerHandlerConfig) (http.
 
 // formatForwardResponseError converts an error to a status code and writes the code to a response.
 func (s *signerHandler) formatForwardResponseError(rw http.ResponseWriter, r *http.Request, err error) {
-	s.Log.WithError(err).Debugf("Failed to process request.")
+	s.Log.With("error", err).DebugContext(s.closeContext, "Failed to process request.")
```

Comment:
Nit: I've been trying to avoid using `With` where the returned logger isn't reused, and instead just specifying attributes at the end of the `DebugContext` call. I'm unsure if that's something other's have been trying to avoid as well.

Reply:
Seems good, I'll try to convert every call 👍